vagrant/setup.sh

```diff
@@ -150,6 +168,7 @@ case ${ID,,} in
         packages+=(nfs-utils nfs-utils-lib)
         ;;
     esac
+    enable_iommu
```

Comment:
how about making the call conditional. e.g., run `enable_iommu` only if it's not enabled and it needs to be enabled.

Reply:
I think that the conditional is included into the function isn't it?

Reply:
I think that the conditional is included into the function isn't it?

Reply:
yes it is. I was suggesting to change for better readability: if (reasons to enable) then call enable_iommu